TESTING.asciidoc

```diff
@@ -489,6 +483,23 @@ supports a remote debugging option:
 gradle run --debug-jvm
 ---------------------------------------------------------------------------
 
+== Debugging remotely from an IDE
+
+If you want to run elasticsearch and be able to remotely attach the process
+for debugging purposes from your IDE, you need to add the following line in
+`config/jvm.options`:
+
+---------------------------------------------------------------------------
+-Xdebug -Xrunjdwp:server=y,transport=dt_socket,address=4000,suspend=n
+---------------------------------------------------------------------------
+
+Then start elasticsearch with `bin/elasticsearch` as usual.
+
+If you are using IntelliJ, create a new Run/Debug configuration, choose `Remote`
```

Comment:
Let's make this more generic and just say consult your IDE documentation for how to attach a debugger to a JVM process.